src/webui/api/torrentscontroller.cpp

```diff
@@ -623,6 +623,21 @@ void TorrentsController::setDownloadLimitAction()
     applyToTorrents(hashes, [limit](BitTorrent::TorrentHandle *torrent) { torrent->setDownloadLimit(limit); });
 }
 
+void TorrentsController::setShareLimitsAction()
+{
+    checkParams({"hashes", "ratioLimit", "seedingTimeLimit"});
+
+    const qreal ratioLimit = params()["ratioLimit"].toDouble();
+    const qlonglong seedingTimeLimit = params()["seedingTimeLimit"].toLongLong();
+    const QStringList hashes {params()["hashes"].split('|')};
```

Comment:
for consistency please change to: `const QStringList hashes = params()["hashes"].split('|');` or you can choose using the curly braces in the above 2 lines instead.

And now its time to squash commits!

I'll turn a blind eye to the coding style comments on JS for now, mind you we also updated the coding guidelines recently (#8629 & #8640).